samples/samples-csharp/TriggerBindingSamples/ReservedPrimaryKeyColumnNamesTrigger.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Extensions.Sql.Samples.Common;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Sql.Samples.TriggerBindingSamples
+{
+    public static class ReservedPrimaryKeyColumnNamesTrigger
```

Comment:
For each of these functions please add a doc comment on the class explaining what scenario it's testing.

Reply:
@Charles-Gagnon, can you explain why we should have different requirements for trigger tests compared to input and output bindings tests?